cmd/minikube/cmd/start.go

```diff
@@ -1539,6 +1540,13 @@ func validateKubernetesVersion(old *config.ClusterConfig) {
 		}
 		exitIfNotForced(reason.KubernetesTooOld, "Kubernetes {{.version}} is not supported by this release of minikube", out.V{"version": nvs})
 	}
+	if nvs.GT(newestVersion) {
+		out.WarningT("Specified Kubernetes version {{.specified}} is more than the newest supported version: {{.newest}}", out.V{"specified": nvs, "newest": constants.NewestKubernetesVersion})
```

Comment:
```suggestion
		out.WarningT("Specified Kubernetes version {{.specified}} is newer than the newest supported version: {{.newest}}", out.V{"specified": nvs, "newest": constants.NewestKubernetesVersion})
```

Reply:
@nishipy I think the goal of this was to prevent accidental Large versions such as 2.5.0
so we should still allow users to try patch versions which are newer,

so maybe we should have a constant in our Code 

Max major version allowed and that would be 1

so if user types --kubernetes-version=2.10.0 it will fail without trying

Reply:
@medyagh Thanks for your comment. I understood our goal.

I think we have two ways to specify the Max major version allowed:
1. define a new constant like `MaxKubernetesMajorVersion = 1`
2. use existing constant (e.g. `NewestKubernetesVersion = "v1.23.2-rc.0"`) and trim the Major version from it. then, we use it as the Max major version allowed.

I think 1st one is better but we have to manage the new constant. On the other hand, 2nd one needs no new constant. What do you think?

Reply:
number 2 seems like a good idea

Reply:
I like the idea of getting the Newest Major version from our NewestKubernetesVersion

Reply:
Thank you @medyagh.
I implemented this flow as following commit: 
9da7049f8ac7b3d33ca6b699da2c99d4f6066653